src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int number_of_balls = 3;
```

Comment:
Constant names should be written in uppercase letters. According to the Java style guide, the name `number_of_balls` should be `NUMBER_OF_BALLS`.